src/vue/__tests__/index.test.ts

```diff
@@ -69,6 +69,30 @@ test('vue $watch', async() => {
   expect(mockCallback.mock.calls.length).toBe(1)
 })
 
+test('test SID if pass event to effector object', async() => {
+  const evt = createEvent()
+  const mockCallback = jest.fn()
+
+
+  const component = createComponent({
+    template: '<div>{{ evt }}</div>',
+    watch: {
+      evt() {
+        mockCallback()
+      }
+    }
+  }, {evt})
```

Comment:
Maybe it's a time to merge both arguments into one config? Like we did in `forest`

```typescript
const component = createComponent({
  template: '<div>{{ evt }}</div>',
  event: {evt},
  watch: {
    evt() {
      mockCallback()
    }
  },
})
```

Reply:
no-no. vue has the same section. it's `methods`.
in this case, we testing, what happened when we pass an event to store object